src/app/qgisapp.cpp

```diff
@@ -6750,45 +6751,47 @@ void QgisApp::fileSaveAs()
     defaultPath += QString( '/' + QgsProject::instance()->title() );
   }
 
-  const QString qgsExt = tr( "QGIS files" ) + " (*.qgs *.QGS)";
-  const QString zipExt = tr( "QGZ files" ) + " (*.qgz)";
-
-  QString exts;
   Qgis::ProjectFileFormat defaultProjectFileFormat = settings.enumValue( QStringLiteral( "/qgis/defaultProjectFileFormat" ), Qgis::ProjectFileFormat::Qgz );
-  switch ( defaultProjectFileFormat )
-  {
-    case Qgis::ProjectFileFormat::Qgs:
-    {
-      exts = qgsExt + QStringLiteral( ";;" ) + zipExt;
-      break;
-    }
-    case Qgis::ProjectFileFormat::Qgz:
-    {
-      exts = zipExt + QStringLiteral( ";;" ) + qgsExt;
-      break;
-    }
-  }
+  const QString qgisProjectExt = tr( "QGIS Project Formats" ) + ( defaultProjectFileFormat == Qgis::ProjectFileFormat::Qgz ? " (*.qgz *.QGZ *.qgs *.QGS)" : " (*.qgs *.QGS *.qgz *.QGZ)" );
+  const QString qgzProjectExt = tr( "QGIS Standard Project Format" ) + " (*.qgz *.QGZ)";
```

Comment:
QGIS Zipped Project Format seems more significant no? Or is QGIS Standard Project Format is something defined somewhere?

Reply:
Just to add -- in my view if we phrase this as something like "qgis project archive" then it can mistakenly convey that it's for storing finished/archived projects.

Reply:
I'm rather -1 to "legacy" for similar reasons. It's not deprecated, and has good use cases such as storing in version control.

Reply:
> Just to add -- in my view if we phrase this as something like "qgis project archive" then it can mistakenly convey that it's for storing finished/archived projects.

Gimp is using the "archive" wording BUT I understand your point.

Why not?
- Package
- Bundle
- Classic

Reply:
@troopa81 , alright, what going for

QGIS Bundled Project (for .qgz)
QGIS XML Project (for .qgs)

I think that'd work.